services/platform/platform.go

```diff
@@ -0,0 +1,200 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// Package platform provides authentication/authorization functionality.
+package platform
+
+import (
+	"context"
+	"net"
+	"os"
+	"time"
+
+	api "github.com/percona-platform/saas/gen/auth/external"
+	"github.com/percona/pmm/utils/tlsconfig"
+	"github.com/percona/pmm/version"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/metadata"
+	"gopkg.in/reform.v1"
+
+	"github.com/percona/pmm-managed/models"
+)
+
+const (
+	defaultHost            = "check.percona.com:443"
+	defaultRefreshInterval = 24 * time.Hour
+	timeout                = 5 * time.Second
+
+	envHost            = "PERCONA_TEST_AUTH_HOST"
+	envRefreshInterval = "PERCONA_TEST_REFRESH_INTERVAL"
```

Comment:
`PERCONA_TEST_AUTH_REFRESH_INTERVAL`. The current name is too generic

Reply:
Changed to `PERCONA_TEST_SESSION_REFRESH_INTERVAL`, long but precise.